### DISCUSSION TITLE
WMW

### DISCUSSION THREAD

**Discussion by sarahhzhangg** [ECCO-Hackweek/ecco-2024#155] [General]
Exploring Southern Ocean water-mass transformations, volume budgets, and how ECCO represents AABW.

**Comment by sarahhzhangg** [comment]
**10/15/24**
What we did: further defined our project; explored ECCO density, temperature, salinity outputs; animated Antarctic surface density over one season
What went well: familiarizing with ECCO; project consultation
What didn't go well: figuring out how to make a volumetric TS plot (3D plot in python)
What we could use help with: how to make a 3d density plot in Python

**Reply by ikeshwani** [reply]
https://github.com/user-attachments/assets/6a40a3a9-cb8d-4044-aca8-f644876e5121

animation for surface density along antarctic coast from sept 1, 1999 - march 31, 2000

**Comment by sarahhzhangg** [comment]
<img width="599" alt="Screenshot 2024-10-16 at 3 46 46 PM" src="https://github.com/user-attachments/assets/c075d1db-60eb-43f1-b432-f455d3c5ecb6">

First take of volumetric TS plot, the range of values seem to be wrong...

**Comment by ikeshwani** [comment]
https://github.com/user-attachments/assets/3af5fd0f-9c2d-47e0-9b04-561b7985fcec

Animation of ocean thickness around Antarctica. Initially at the surface and shows the thickness through 50 vertical layers, moving downwards, until the thickness is zero because we've hit the ocean floor.

**Reply by ikeshwani** [reply]
but something is wrong, the coloring in animation doesn't match up with snapshots of the plots

**Comment by ikeshwani** [comment]
Also, started looking at surface salt flux data to due brine rejection over the period February 2000 - October 2000. Here are two plots comparing the initial and final surface salt flux.  
Initial: Day 0
![download-33](https://github.com/user-attachments/assets/bd711a90-0de0-488f-a4ab-a3d35b3c34fc)

Final- Day 274
![download-34](https://github.com/user-attachments/assets/ff66c7ae-ae78-46b3-9fe2-4833c62cb31f)

We are working on an animation.

**Comment by ikeshwani** [comment]
Also here's an updated video of the surface density over the period february 2000 to october 2000. The surface gets denser over the period!

https://github.com/user-attachments/assets/f68b1687-7cbf-49e2-9734-f12c8fb3e33f

**Comment by sarahhzhangg** [comment]
Our final poster!
[ECCO Hack Week Poster.pptx](https://github.com/user-attachments/files/17440083/ECCO.Hack.Week.Poster.pptx)
